src/components/testCaseLanding/qiCore/TestCaseList.tsx

```diff
@@ -254,6 +254,30 @@ const TestCaseList = (props: TestCaseListProps) => {
     }
   };
 
+  const exportTestCases = async () => {
+    const textFile = new Blob([JSON.stringify("")], {
```

Comment:
can we call the stubbed out endpoint in the measure-service as noted on the JIRA

Reply:
basically, with the completion of this story, when the button is clicked is should call the backend API and download the zip returned from the backend. For now that backend file will be empty as it's a stubbed out API, but the frontend needs to talk to the backend here. Otherwise we'd need to create another story to complete the functionality and reduce the original scope and points of this story.

Reply:
@nmorasb API call is added, thanks Nate!